addons/account/wizard/accrued_orders.py

```diff
@@ -167,6 +172,7 @@ def _get_aml_vals(order, balance, amount_currency, account_id):
                     inc_exp_accounts[account]['amount_currency'] += amount_currency
                     total_balance += amount
                     total_amount_currency += amount_currency
+                self.invalidate_cache()
```

Comment:
comment this please :)

Reply:
@rco-odoo FYI: we had to do that, might be worth checking if that's a problem in the cache management of the ORM, or just us doing stupid things stupidly